frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/PullApprovalDialog.kt

```diff
@@ -0,0 +1,20 @@
+package com.virtuslab.gitmachete.frontend.actions.dialogs
+
+import com.intellij.ide.util.PropertiesComponent
+import com.intellij.openapi.ui.DoNotAskOption
+import com.intellij.openapi.ui.Messages
+import com.virtuslab.gitmachete.frontend.actions.traverse.TraverseRepositoryAction
+import com.virtuslab.gitmachete.frontend.resourcebundles.GitMacheteBundle.getString
+
+class PullApprovalDialog : DoNotAskOption.Adapter() {
+
+  override fun rememberChoice(isSelected: Boolean, exitCode: Int) {
+    if (exitCode == Messages.OK && isSelected) {
+      PropertiesComponent.getInstance().setValue(TraverseRepositoryAction.SHOW_PULL_APPROVAL, false, /* defaultValue */ true)
+    }
+  }
+
+  override fun getDoNotShowMessage(): String {
```

Comment:
do we really need to override this one? what's the original message?

Reply:
🤔 we had the same functionality in the `WranAboutSyncToParentByMergeDialog`
Though I might as well need to remove the override to see how it behaves by default.

Reply:
yeah, probably there is no need to have it there too. just make sure that the default message is provided and sounds reasonable